prisma/migrations/20211225043237_add_transaction_table/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "Transaction" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    "userId" TEXT NOT NULL,
+    "subscriptionId" TEXT NOT NULL,
+    "bankName" TEXT NOT NULL,
+    "bankAccountNumber" TEXT NOT NULL,
+    "bankAccountName" TEXT NOT NULL,
+    "amount" INTEGER NOT NULL,
+    "status" TEXT NOT NULL,
```

Comment:
Also mas @zainfathoni , do you handle multiple channel for payment? If yes it always recommend to save the `vendor` name like it will be like `midtrands`, `oy`, or maybe `manual`. Sometimes I just want to use the cheapest vendor of my choice.

Reply:
It sounds good to me. However, since we only handle **manual** transactions at the moment, I don't think it's necessary. We don't know which payment methods we are willing to support in the future, so that I would consider any assumptions on that area as premature optimization. 😅

It's a good idea, though. So let's keep it in a separate issue in the repository. Would you please propose it as a GitHub issue, @mandaputtra? 😁 Thanks! 🙏

Reply:
FYI, Mas @mandaputtra. I just incorporated your suggestion in #86. 😁
Thanks for your advice! 🙏